.env

```diff
@@ -27,6 +28,7 @@ JWT_SECRET=testsecret
 SERVICE_ID=test_service_id
 
 # PUBSUB Emulator
+PUBSUB_PROJECT=project
```

Comment:
what's this for? The projects should be either `our-project` or `shared-project`

Reply:
I think this can be removed